index.html

```diff
@@ -744,15 +740,55 @@ <h4>On Screen</h4>
   | "OFF_ON"
                   </pre>
                 </div>
-                <p class="issue" data-number="34"></p>
-              </li>
-              <li>A <code>ttm:desc</code> element may be present representing the <a>Script Event Description</a>.</li>
-              <li><a>Script Event Type</a>
-                <p class="issue" data-number="35"></p>
               </li>
             </ul>
           </p>
         </section>
+        <section>
+          <h4>Script Event Description</h4>
+          <p>The <dfn>Script Event Description</dfn> property is an annotation providing a human-readable description of a <a>Script Event</a>.
+          <p>The <a>Script Event Description</a> property is represented in TTML as <code>ttm:desc</code> element added to a <code>metadata</code> element at the <code>div</code> level.</p>
+          <p class="note">The <a>Script Event Description</a> does not need to be unique, i.e. it does not need to have a different value for each <a>Script Event</a>. 
+            For example a particular value could be re-used to identify in a human-readable way one or more <a>Script Events</a> that are intended to be processed together,
+            e.g. in a batch recording.</p>
+        <pre class="example"
+          data-include="examples/event-desc.xml"
+          data-include-format="text">
+        </pre>
```

Comment:
This example is so similar to the following one with `descType` that I don't think we need both. Suggest removing this and the file, unless I've missed something.

Reply:
I expect the example with no descType to be the most frequent case. I'd rather keep it.

Reply:
Can we simplify the non-descType example then, to include just a `<div>`, and maybe take out the `<metadata>` element?

Reply:
Oh, I see... `ttm:desc` is in the `Metadata.class` and can be put directly under the `div` without needing the `metadata` element. I can fix that.
Regarding simplification of the non-descType, the idea of the example is to show that the same description can be used for 2 consecutive events. I'm not sure it makes sense to remove the content.
I think we're struggling here because the use of `descType` is not clear. Maybe describing more the second example to say something like: "This example is equivalent to the previous one except that the actual type of event description is given when disambiguation is needed".

Reply:
Sure, two different examples are fine, but not so similar. How about something like:

ttm:desc non-uniqueness:

```xml
<div [attributes] daptm:onScreen="OFF">
  <ttm:desc>Voiceover</ttm:desc>
  <p daptm:langSrc="original">Will she remember how to sail, after all these years?</p>
</div>

<div [attributes] daptm:onScreen="OFF">
  <ttm:desc>Voiceover</ttm:desc>
  <p daptm:langSrc="original">Of course!</p>
</div>
```

ttm:desc with descType:

```xml
<div [attributes]>
  <ttm:desc daptm:descType="pronunciationNote">[oːnʲ]</ttm:desc>
  <p daptm:langSrc="original">Eóin looks around at the other assembly members.</p>
</div>
```